routes/api.php

```diff
@@ -37,6 +37,9 @@ function ($router) {
         $router->delete('blocks/{id}', 'BlockController@destroy');
         $router->get('blocks/{id}', 'BlockController@show');
 
+        $router->get('blocks/{id}/files', 'BlockFileController@index');
+        $router->post('blocks/{id}/files', 'BlockFileController@store');
```

Comment:
Since after sync only the IDs in the given array will exist in the intermediate table, I think that `PUT` method will be more accurate here.

A PUT is a complete replacement that ignores the current state of the resource.

Maybe `BlockFileController@sync`?

Reply:
Ok, it is done now.